runtime/shared_common/j9shr.tdf

```diff
@@ -2938,3 +2938,26 @@ TraceEvent=Trc_SHR_CM_updateLocalHintsData_WriteHeapSizes Overhead=1 Level=4 Tem
+TraceEntry=Trc_SHR_CC_initializeWithCommonInfo_Entry2 NoEnv Overhead=1 Level=3 Template="CC initializeWithCommonInfo: memForConstructor=%p, ctrlDirName=%s, cacheName=%s, newPersistentCacheReqd=%d, layer=%d"
+TraceEntry=Trc_SHR_CC_initialize_Entry2 NoEnv Overhead=1 Level=1 Template="CC initialize: memForConstructor=%p, sharedClassConfig=%p, cacheName=%s, newPersistentCacheReqd=%d, layer=%d, unitTest=%zd"
+TraceEntry=Trc_SHR_OSC_newInstance_Entry_V1 NoEnv Overhead=1 Level=1 Template="SH_OSCache::newInstance: Entered, memForConstructor=%p, cacheName=%s, newPersistentCacheReqd=%d layer=%d"
+
+TraceException=Trc_SHR_CM_getPrereqCache_InvalidType Overhead=1 Level=1 Template="CM::getPrereqCache(): Invalid data type found in the shared cache at %p. The shared cache is corrupted"
+TraceEvent=Trc_SHR_CM_getPrereqCache_Found Overhead=1 Level=6 Template="CM::getPrereqCache(): Found prerequisite cache with unique ID %.*s"
+TraceEvent=Trc_SHR_CM_getPrereqCache_NotFound Overhead=1 Level=1 Template="CM::getPrereqCache(): This shared cache does not have a prerequisite cache"
+TraceException=Trc_SHR_CM_getPrereqCache_Failed_To_Start_Manager Overhead=1 Level=1 Template="CM::getPrereqCache(): Failed to startup the corresponding manager"
+TraceException=Trc_SHR_CM_getPrereqCache_Failed_To_Add_ID Overhead=1 Level=1 Template="CM::getPrereqCache(): Failed to add unique ID of prerequisite cache"
+TraceException=Trc_SHR_CM_getPrereqCache_Failed_To_Get_Manager Overhead=1 Level=1 Template="CM::getPrereqCache(): Failed to get the corresponding manger"
+TraceException=Trc_SHR_CM_getPrereqCache_Failed_To_Store_Prereq_UniqueID Overhead=1 Level=1 Template="CM::getPrereqCache(): Failed to store the prerequisite cache unique ID %.*s"
+TraceException=Trc_SHR_CC_verifyCacheUniqueID_Failed Overhead=1 Level=1 Template="CC verify Cache Unique ID Failed. The Expected ID is %s, the current ID is %s"
+TraceExit-Exception=Trc_SHR_CM_startup_Exit8 Overhead=1 Level=1 Template="CM startup: Cache Unique ID Verification Failed"
+TraceExit-Exception=Trc_SHR_CM_startup_Exit9 Overhead=1 Level=1 Template="CM startup: Failed to get prerequisite unqiue cache ID, error is %d"
+TraceExit-Exception=Trc_SHR_CM_startup_Exit10 Overhead=1 Level=1 Template="CM startup: Failed to allocate buffer for SH_CompositeCacheImpl"
+TraceExit-Exception=Trc_SHR_CM_startup_Exit11 Overhead=1 Level=1 Template="CM startup: Failed to start up the shared cache"
+TraceExit-Exception=Trc_SHR_CM_startup_Exit12 Overhead=1 Level=1 Template="CM startup: Failed to allocate memory for the cache descriptor"
+TraceExit-Exception=Trc_SHR_CM_startup_Exit13 Overhead=1 Level=1 Template="CM startup: Failed to read the cache. Cache is corrupted"
+TraceExit=Trc_SHR_CM_updateROMClassResource_Exit7 Overhead=1 Level=1 Template="CM updateROMClassResource: The address to be updated %p (length %zu) is not in top layer cache."
```

Comment:
I'm wondering if this should be a TraceExit-Exception since this case shouldn't be occurring.

Reply:
Changed to TraceExit-Exception.